templates/layout-product-variation-slider.php

```diff
@@ -3,11 +3,6 @@
 
 $show_page_dots = FALSE;
 $slider_image_size = has_image_size('woocommerce_thumbnail') ? 'woocommerce_thumbnail' : 'medium';
-// Prevent wrong images height calculation caused by lazy loading.
-$image_attr = apply_filters('gallerya_lazyload_image_attributes', [
```

Comment:
Why is this obsolete now?

Reply:
Good question. I thought it wasn't needed anymore, because we're using Flickity for the lazy-loading here. But when thinking about it now, probably the first image is still handled by BJ Lazy Load and this code is still needed. What do you think?

Reply:
@fabianmarz This would lead to the first images not being lazy loaded though – sometimes there are lots of variation thumbnail sliders in a product grid.  This would be bad for performance, I don't think this makes sense.

Reply:
I think this should be removed, otherwise, as @moritzbappert said, the lazy load will not work on the gallerya images.

But, we should double check gallerya first image is still displayed correctly also for regular post galleries, not only for products thumbnails.

Reply:
I have removed this only for this particular case though, the variation sliders in the product listings grids, just to make sure this is understood correctly.